src/js/injectBaseStylesheet.js

```diff
@@ -63,5 +63,6 @@ export default function injectBaseStylesheet() {
   const head = document.head;
 
   const allHeadElements = head.getElementsByTagName("*");
-  allHeadElements.innerHTML = styleEl + allHeadElements.innerHTML;
+  const styleTag = allHeadElements.item(allHeadElements.length - 1);
```

Comment:
```suggestion
  const styleTag = allHeadElements.item(0);
```

We want to select the first element, not the last one (afaik)